h2o-r/tests/testdir_algos/rulefit/runit_rulefit_titanic.R

```diff
@@ -31,7 +33,7 @@ test.rulefit.titanic <- function() {
     expect_that(h2o.auc(h2o.performance(rfit)), equals(h2o.auc(h2o.performance(rfit, newdata =  train))))
     expect_that(h2o.logloss(h2o.performance(rfit)), equals(h2o.logloss(h2o.performance(rfit, newdata =  train))))
 
-    rfit <- h2o.rulefit(y = response, x = predictors, training_frame = train, validation_frame = test, min_rule_length = 1, max_rule_length = 10, max_num_rules = 100, seed = 1, model_type="rules")
+    rfit <- h2o.rulefit(y = response, x = predictors, training_frame = train, validation_frame = test, min_rule_length = 1, max_rule_length = 10, max_num_rules = 100, seed = 1, model_type="rules", lambda=0.5)
```

Comment:
I am missing some assert, the lambda was applied. The same for pyunit. Maybe train a model with lambda is set and compare it with a model where lambda is not set? Does it make sense?

Reply:
I think it makes sense to check that the number of important rules differs for different lambdas 👍

Reply:
c0f9db5